test/e2e/scenarios/add_cluster.go

```diff
@@ -0,0 +1,62 @@
+package scenarios
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = PDescribe("[Dataplane] Adding existng network policy to newly added cluster", func() {
+	PContext("Registering cluster 3 with kubefed and then creating a pod", func() {
+		PIt("Should implement existng network policy in newly added cluster", func() {
+			By("creating listener pod 1 with label 1 in cluster 1 in namespace 1")
```

Comment:
Assuming that this test-case validates use-case number#2 in the document, namespaces (i.e., boring-project) should all be the same right?

CC: @mangelajo

Reply:
The policy is namespace selector based and hence different namespaces. I will specify this in the description.